services/classic/management/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Service Management packages for Go
 
-The `github.com/Azure/azure-sdk-for-go/services/classic/management` packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](../arm) folder.
+The `github.com/Azure/azure-sdk-for-go/services/classic/management` packages are used to perform operations using the Azure Service Management (ASM), aka classic deployment model. Read more about [Azure Resource Manager vs. classic deployment](https://azure.microsoft.com/documentation/articles/resource-manager-deployment-model/). Packages for Azure Resource Manager are in the [arm](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/armcore) folder.
```

Comment:
Link should be https://github.com/Azure/azure-sdk-for-go/tree/master/services

Reply:
And also, should this be `services` folder?

Reply:
@ArcturusZhang, Checked in your suggestions.